Client/sdk/game/Common.h

```diff
@@ -1548,6 +1548,25 @@ enum eObjectProperty
     OBJECT_PROPERTY_MAX,
 };
 
+enum eShadowType
+{
+    // use only color instead of texture
+    SHADOW_PLANE = -1,
+    SHADOW_CAR,
+    SHADOW_PED,
+    SHADOW_HELI,
+    SHADOW_BIKE,
+    SHADOW_RCBARON,
+    SHADOW_EXPLOSION,
+    SHADOW_HEADLIGHT1,
+    SHADOW_HEADLIGHT2,
+    SHADOW_BLOOD,
+    SHADOW_HEADMAN,
+    SHADOW_WINCRACK,
+    SHADOW_LAMP,
+    SHADOW_COUNT,
+};
```

Comment:
```suggestion
enum class eShadowType
{
    // use only color instead of texture
    PLANE = -1,
    CAR,
    PED,
    HELI,
    BIKE,
    RCBARON,
    EXPLOSION,
    HEADLIGHT1,
    HEADLIGHT2,
    BLOOD,
    HEADMAN,
    WINCRACK,
    LAMP,
    _COUNT,
};
```